field_audit/models.py

```diff
@@ -568,13 +573,42 @@ def delete(self, *, audit_action=AuditAction.RAISE):
         return value
 
     @validate_audit_action
-    def update(self, *args, audit_action=AuditAction.RAISE, **kw):
+    def update(self, audit_action=AuditAction.RAISE, **kw):
         if audit_action is AuditAction.IGNORE:
-            return super().update(*args, **kw)
-        else:
-            raise NotImplementedError(
-                "Change auditing is not implemented for update()."
+            return super().update(**kw)
+        assert audit_action is AuditAction.AUDIT, audit_action
+        from .field_audit import request
+        request = request.get()
```

Comment:
This can be moved after the "noop short-circuit" since it's only needed if you actually need to create audit events.

Reply:
See 866af4c7bb877f02271a50cc55b9a7aadf9857c1